src/dialects/postgres/schema/tablecompiler.js

```diff
@@ -41,6 +41,52 @@ TableCompiler_PG.prototype.createQuery = function(columns, ifNot) {
   if (hasComment) this.comment(this.single.comment);
 };
 
+TableCompiler_PG.prototype.addColumns = function(columns, prefix, colCompilers) {
+  if (prefix === this.alterColumnsPrefix) {
+    // alter columns
+    for (const col  of colCompilers) {
+      const quotedTableName = this.tableName();
+      const colName = col.getColumnName();
```

Comment:
Are there guidelines on aligning consecutive assignments? If so, apply here and in object literals.